### DISCUSSION TITLE
Strommenge eines Gerätes von anderem Gerät abziehen

### DISCUSSION THREAD

**Discussion by MarLuRi** [camueller/SmartApplianceEnabler#500] [Installation und Betrieb]
Hallo in die Runde und an dich Axel,
Ist es möglich die gemessene Strommenge eines Gerätes von einem anderen in SAE hinterlegten Gerät abzuziehen? 

Folgendes Szenario ist bei mir geplant bzw. teilweise umgesetzt:
Jeder Unterverteiler im Haus hat einen separaten Zähler mit S0-Schnittstelle. Diese werden auch im SHM und der App angezeigt. 
Im Unterverteiler ‚Keller‘ befindet sich zusätzlich noch ein dreiphasiger Heizstab (3x2KW), welcher über  drei Tasmota (Http) geschalten wird. Nun verfälscht dieser Wert aber die Grafik sowie Abrechnungsmodalitäten Hausintern, da sowohl ‚Keller‘ als auch ‚Heizstab‘ gezählt werden. Wir verbrauchen derzeit auf dem Papier bspw. 12KW, obwohl es nur 6KW sind.. 

Wäre es daher möglich, von dem Gerät ‚Keller‘ den tatsächlichen Verbrauch des Gerätes bzw der Geräte ‚Heizstab’ abzuziehen und den bereinigten Wert ‚Keller’ an den SHM zu senden?

Besten Dank!

**Comment by camueller** [comment]
Nein, das ist nicht möglich. Im SAE werden nur "Geräte" verwaltet. "Keller" ist kein Gerät, sondern eine Sammlung von Geräten.
